client/deployment/src/main/resources/templates/libraries/microprofile/pojo.qute

```diff
@@ -10,7 +10,16 @@ import jakarta.xml.bind.annotation.XmlEnumValue;
 import java.lang.reflect.Type;
 import com.fasterxml.jackson.annotation.JsonProperty;
 {/if}
+{#if use-bean-validation}
+import jakarta.validation.constraints.*;
+import jakarta.validation.Valid;
```

Comment:
To avoid [compilation errors due to ambiguity](https://github.com/quarkiverse/quarkus-openapi-generator/issues/103), can you please use the FQCN rather than adding imports?